libexec/swiftenv-install

```diff
@@ -346,6 +361,13 @@ VERSION="${VERSION##swift-}"
 
 check_installed "$VERSION"
 
+if ! $user; then
+  if [ "$(uname)" != "Darwin" ]; then
+    echo "--no-user installation is only supported on macOS."
```

Comment:
`--user installation` ?

Reply:
This branch will only happen if you supply `--no-user` on a platform other than Darwin (windows, bsd, linux etc). Those platforms currently only support "user" install which is into `$SWIFTENV_ROOT/versions` (usually `~/.swiftenv/versions`).